json/wireguard.json

```diff
@@ -9,7 +9,7 @@
     "updateable": true,
     "privileged": false,
     "interface_port": "10086",
-    "documentation": null,
+    "documentation": "https://github.com/community-scripts/ProxmoxVE/wiki/WireGuard-vs.-WGDashboard",
```

Comment:
Not sure this should be on our wiki, wouldn't that be better maintained on `WGDashboard` github?

Reply:
our script spams the creator 😄 
https://github.com/community-scripts/ProxmoxVE/issues/245#issuecomment-2477085138

Reply:
Adding a note is fine, but this is a full wiki page explaining the difference between Wireguard and WGDashboard, which we have to maintain and support. Perhaps the WGDashboard creator can have it in that repo instead and we can link to it there. I don't think this is in community scripts scope to have a wiki page about.
Also `documentation` button should probably link to Wireguard docs instead

Reply:
Then we need to Link @donaldzou Here, or you give Feedback in issue

Reply:
Hi! Sure we can host one over our side, do you mind if I just post what you had in the Wiki and we'll maintain from there?

Reply:
> our script spams the creator 😄 [#245 (comment)](https://github.com/community-scripts/ProxmoxVE/issues/245#issuecomment-2477085138)

LOL not really spamming but we just gotta also answer WireGuard questions on WGDashboard repo.. which is ok but we want users to know is not the same thing haha

Reply:
yep, fine for us. give feedback if you done, then i update the PR and merge it

Reply:
@MickLesk Maybe the documentation button should link to https://www.wireguard.com/quickstart/ and we can have the WGDashboard documentation link on the note you added? 
If the LXC documentation button links to `WGDashboard` people might think that is where they should ask wireguard questions.

Reply:
Yep, after Donald response, i change documentation link and add a note

Reply:
Just finished building the site: https://donaldzou.github.io/WGDashboard-Documentation/what-is-wireguard-what-is-wgdashboard.html

Reply:
I'll update it after lunch👍

Reply:
Thanks for the quick response regarding this :)